xdocs/building.xml

```diff
@@ -109,22 +109,20 @@ Check that patch does not break JUnit tests by running:
 </p>
 <source>./gradlew test</source>
 
-<h3>Create a PR using GIT</h3>
+<h3>Create a pull request using Git</h3>
 <ul>
-    <li>Open a bugzilla issue, see <a target="_blank" href="issues.html">Issues page</a></li>
-    <li>Fork <a href="https://www.github.com/apache/jmeter" >Apache JMeter mirror</a></li>
-    <li>Clone your forked repository locally : <source>git clone https://github.com/yourid/jmeter/</source></li>
-    <li>Create a branch using for example bugzilla id: <source>git branch BUGID</source></li>
-    <li>Checkout the new branch : <source>git checkout BUGID</source></li>
-    <li>commit your fix there : <source>git commit -m 'Fix to BUGID' list of files</source></li>
+    <li>Fork <a href="https://www.github.com/apache/jmeter">Apache JMeter mirror</a></li>
+    <li>Clone your forked repository locally: <source>git clone https://github.com/yourid/jmeter.git</source></li>
+    <li>Create a branch using for example issue id: <source>git branch gh123-thread-group-typo</source></li>
+    <li>Checkout the new branch: <source>git checkout gh123-thread-group-typo</source></li>
+    <li>Commit your fix there: <source>git commit -m 'Fix to BUGID' list of files</source></li>
     <li>Please avoid creating merge commits in the PR. We embrace small changes, and merge commits are harder to review</li>
-    <li>Add JMeter repository as upstream : <source>git remote add upstream http://www.github.com/apache/jmeter</source></li>
-    <li>push it : <source>git push origin BUGID</source></li>
-    <li>Create a PR and link it in the bugzilla issue</li>
-    <li>Or create a diff and add as attachment to bugzilla issue <source>git diff &gt; BUGID.patch</source></li>
+    <li>Add JMeter repository as upstream: <source>git remote add upstream https://www.github.com/apache/jmeter.git</source></li>
+    <li>Push it: <source>git push upstream gh123-thread-group-typo</source></li>
```

Comment:
I have never tried this, but does it work for all users?

Reply:
Of course, it will not. It should suggest pushing to the fork repository rather than upstream one.